shell/packages/sandstorm-db/db.js

```diff
@@ -1875,6 +1891,77 @@ _.extend(SandstormDb.prototype, {
 
     return quota && user.storageUsage && user.storageUsage >= quota.storage * 1.2 && "outOfStorage";
   },
+
+  suspendIdentity: function (userId, suspension) {
```

Comment:
Please add parameter check()s for all methods in SandstormDb, for extra protection.